docs/pages/custom-builds/schema.mdx

```diff
@@ -770,6 +770,38 @@ build:
   href="https://github.com/expo/eas-build/blob/main/packages/build-tools/src/steps/functions/installNodeModules.ts"
 />
 
+#### `eas/get_credentials_for_build_triggered_by_github_integration`
+
+Resolves the credentials for a build triggered by our [GitHub integration](/build/building-from-github/). If you are using the GitHub integration and your custom build needs to use credentials, you must call this function.
```

Comment:
```suggestion
Resolves the credentials for a build triggered by our [GitHub integration](/build/building-from-github/). Call this function if you are using the GitHub integration and your custom build needs to use credentials.
```

- Let's be more authoritative in the last sentence about when to use this function.